### ISSUE TITLE
[Time field] Active options in expandable Time picker do not have sufficient contrast ratio in Dark mode

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3027]
### Figma 
There is no Dark mode version of Time Picker with disabled minutes options. 

### Storybook 
All active hours and minutes options inside expandable [Time picker](https://storybook.sanomalearning.design/?path=/story/form-time-field--basic&globals=mode:dark) have text color #000000 on background in color #252930. It results in contrast ratio between text and background of 1.4:1.

<img width="171" height="421" alt="Image" src="https://github.com/user-attachments/assets/fa885473-1299-46f1-8576-5edad068c7ec" />

### Documentation page
All active hours and minutes options inside expandable [Time picker](https://sanomalearning.design/categories/components/time-field/usage/) have text color #000000 on background in color #252930. It results in contrast ratio between text and background of 1.4:1.

<img width="162" height="432" alt="Image" src="https://github.com/user-attachments/assets/24267fb9-6fc1-4bea-8e3f-bd05cd2e9bd2" />

## Proposed solution
For Figma Dark mode version of Time picker with disabled minutes options should be added.

For Storybook and Documentation page active hours and minutes options color #CCCCCC should be used as it is presented in Figma. Contrast ratio of options text and background will be 9.8:1 then.
When Figma view for Dark mode version of Time picker with disabled minutes options would be ready it should be also implemented in Storybook.

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS
